### PR TITLE
Accountability station pinning

### LINKED ISSUE #7: Pinning and unpinning messages based on reaction
If the author of a message reacts to their own message in #The-Accountability-Club with the push-pin emoji it should be pinned.

If they remove the reaction, the message should be unpinned.

### PR DESCRIPTION
**Pull request by Dendrobyte**
Resolves #7

### CODE REVIEW

**Comment by Dendrobyte** [comment]
Any and all feedback is accepted :)

File: commands/accountabilityUnpin.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	if(message.channel.id === config.channels.accountability) {
+		// Call to method elsewhere
+		accountabilityActions.userUnpinsMessage(message, message.author);
+	}
+};
+
+module.exports.config = {
+	name: 'accountabilityUnpin', // Chose this name for organizational sake
+	aliases: ['accountabilityUnpin', 'unpin'],
+	description: 'I will unpin a message for you in accountability',
```
Comment (Moesgaarda) [inline commands/accountabilityUnpin.js]: Is it a random message? Or your latest pinned?

File: events/messageReactionAdd.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
 const tosActions = require('../eventActions/tosActions');
+const accountabilityActions = require('../eventActions/accountabilityActions');
 const pinAction = require('../eventActions/pinAction');
 
 module.exports = async (client, reaction, user) => {
 	// Handle reaction to the ToS message in ToS channel
 	tosActions.userAcceptsTOS(reaction, user, client);
+
+	// Handle reaction to a message in accountability cahnnel
```
Comment (Moesgaarda) [inline events/messageReactionAdd.js]:
```suggestion
	// Handle reaction to a message in accountability channel
```

File: commands/accountabilityUnpin.js (hunk 1 above)
Comment (Dendrobyte) [reply commands/accountabilityUnpin.js]: It's the latest one- I'll go ahead and update that description to be more specific.

To specify, it goes through the list starting with the most recent pin and removes the first one it finds from a user. Convenient!

File: events/messageReactionAdd.js (hunk 2 above)
Comment (Dendrobyte) [reply events/messageReactionAdd.js]: English 100

**Comment by Dendrobyte** [comment]
> 
> 
> Shouldn't it unpin a message if the author un-reacts to it with the pin?
> Should be as easy as adding the messageReactionRemove (or delete?) event.

As I remember Horace actually removed your pin reaction and gave it his own, but I will definitely go ahead and add this. Forgot about that! Will shoot for Wednesday/Thursday morning commit.

Thanks for comments!

**Comment by Dendrobyte** [comment]
Updated!
- `monkclub` and aliases added to the `!info!` command
- Typo fixed
- Description updated for unpinning latest message with command
- **Can now remove pinned messages by removing your pin react**

**Pull request merged**